### PR TITLE
*: move metric `neogo_version` out of pkg/network

### LINKED ISSUE #3682: Move neogo_version metric out of pkg/network
```
apollo1% curl -s localhost:9901 | grep neogo_version
# HELP neogo_version NeoGo version
# TYPE neogo_version gauge
neogo_version{version=""} 1
```

### PR DESCRIPTION
**Pull request by AliceInHunterland**
Close #3682

### CODE REVIEW

**Review by AnnaShaleva — COMMENTED** [review]
But why? It should be possible to move only the `neogo_version` metric itself, without massive packages move.

File: pkg/network/prometheus.go [hunk 1]
```diff
@@ -72,7 +64,7 @@ func init() {
 	prometheus.MustRegister(
 		estimatedNetworkSize,
 		peersConnected,
-		neogoVersion,
+		network.NeogoVersion,
```
Comment (AnnaShaleva) [inline pkg/network/prometheus.go:67]: As far as metric registration code. The metric itself should be registered inside `cli/server`. Our main goal is not to show this metric for NeoFS node/Morph node.

**Review by AnnaShaleva — COMMENTED** [review]
@AliceInHunterland, the code itself looks as expected. Have you tested it, is it ready for review?

**Comment by AliceInHunterland** [comment]
> Have you tested it

in progress

**Comment by AliceInHunterland** [comment]
Checked!

**Review by roman-khimov — APPROVED** [review]

File: cli/server/server.go [hunk 2]
```diff
@@ -506,7 +506,7 @@ func startServer(ctx *cli.Context) error {
 	errChan := make(chan error)
 	rpcServer := rpcsrv.New(chain, cfg.ApplicationConfiguration.RPC, serv, oracleSrv, log, errChan)
 	serv.AddService(rpcServer)
-
```
Comment (AnnaShaleva) [inline cli/server/server.go:509]: I'd say it's useless change, but OK.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**